hexkit/protocols/dao.py

```diff
@@ -184,6 +173,21 @@ class DaoSurrogateId(DaoCommons[Dto], typing.Protocol[Dto, DtoCreation_contra]):
      creation of new resources, is needed.
```

Comment:
"is needed" -> i**f** needed?

Reply:
Don't know what was written there :D deleted it